guidelines/program-guidelines.md

```diff
@@ -0,0 +1,81 @@
+
+Visit the venue at least once before the event, and prep the volunteers on the basics (especially if they are also coming in from out of town).
+
+## Things take time
+
+Although we “know” that things take time, we rarely schedule time for all the things, *beyond talks*,  that need to happen during an event.  Some of these things include:
+
+  - Applause (seriously)
+  - Plugging/Unplugging Laptops
+  - Speaker Introductions
+  - Announcements
+
+In order to realistically design your program, you should specifically set aside time to account for these things. We recommend:
+
+  - 10min at the beginning and end of the program day
+  - 5-10min between talks
+
+These times serve 2 purposes:
+
+  - realistically set time aside for things that have to happen and things **take time** 
+  - You can use this time to recover speakers going over and under!
+
+You should not only set this time aside but you should clearly show it on the program you display and distribute to attendees. They appreciate a realistic schedule and will be more likely to comply with it if they know when it is and why it’s there in advance which always makes things run more smoothly.
+
+## Tech Check
+It is extremely useful to have your A/V or organizers available to help speakers set up their machines on stage to test their display *before* they have to speak. Inevitably, not every speaker will be available (and some will flake out and forget) but every person you are able to tech check saves you time in your program and also allows the event to appear more professional. 
+
+We are all developers for the most part, and are aware of how computers are terrible and never work. Nevertheless we are an unforgiving group when it appears that people have not prepared for or are surprised that computers 
+
+## Q&A
+We don’t recommend opening the floor to Q&A. We do recommend:
+
+  - Dedicated room/Speaker Corners
+  - Keep it short, speakers want to go to talks too!
+
+## Talk Skill Level
+
+Ask applicants for the kind of audience their talk targets. Are their audience beginners, intermediate or advanced users? How technical is their talk?
+
+The program should try to balance the talks to not overwhelm the beginners and to not bore people familiar with the language with details they already know. Some possible ideas here: 
+
+- Have two tracks running in parallel: one for beginners and one for intermediate / advanced users.
+- Another option is to schedule the beginner talks at the beginning so the that part of the audience feels more confident to attend the more advanced talks.
+
+## Human Limits
+
+Be aware of the human limitations of your participants! People get sleepy after lunch, or they may be hungover on the second morning of the event. An idea here is to have more technical talks after the coffee break and less tech heavy talks after lunch and on the second morning of the event.
+
+## How long and how often should breaks be
+
```

Comment:
The "hallway track" and the breaks are often the most useful parts of a conference. We could use this section to highlight that.